rockspec/lua-tinyyaml-0.3-0.rockspec

```diff
@@ -1,5 +1,6 @@
 package = "api7-lua-tinyyaml"
 version = "0.3.0-0"
+
```

Comment:
Do we need to update the file name `rockspec/lua-tinyyaml-0.3-0.rockspec` to `rockspec/lua-tinyyaml-0.3.0-0.rockspec`?

![image](https://user-images.githubusercontent.com/6814606/125015897-fda98080-e0a2-11eb-856e-637a3aa09cd5.png)

Reply:
ping @Yiyiyimu

Reply:
Thanks for reminding!

@membphis it would be great if you could remove the previous release and tag, so the release actions could re-generate them and move on to upload to luarocks. 
(I tried to upload to luarocks by ignoring release generation failure - since it's already existed - but failed with very weird error. So I guess maybe regenerate release could fix it)